chsdi/templates/htmlpopup/lubis.mako

```diff
@@ -217,7 +211,17 @@ shop_url = request.registry.settings['shop_url']
 
     <tr><th class="cell-left">${_('tt_lubis_originalsize')}</th>     <td>${c['attributes']['originalsize'] or '-'}</td></tr>
     <tr><th class="cell-left">${_('tt_lubis_filesize_mb')}</th>      <td>${filesize_mb or '-'}</td></tr>
-    <tr><th class="cell-left">${_('tt_lubis_bildpfad')}</th>         <td>${filename or '-'}</td></tr>
+    <tr><th class="cell-left">${_('tt_lubis_bildpfad')}</th>
+
+% if filename and filename.startswith('http'):
+        <td><a href="${filename or '-'}" target="_blank">${filename or '-'}</a></td></tr>
+% else:
+        <td>${filename or '-'}</td></tr>
+% endif
+
+% if 'doi_link' in c['attributes'] and c['attributes']['doi_link'] and c['attributes']['doi_link'].startswith('http'):
```

Comment:
this could be written as: `if c['attributes'].get('doi_link', '').startswith('http'):`

Reply:
thanks good point!